go.mod

```diff
@@ -53,7 +53,10 @@ require (
 	knative.dev/serving v0.35.1-0.20221114131921-874ccebb8063
 )
 
-require github.com/sabhiram/go-gitignore v0.0.0-20210923224102-525f6e181f06
+require (
+	github.com/gorilla/mux v1.8.0
```

Comment:
https://github.com/gorilla#gorilla-toolkit

not sure if we should depend on this package

Reply:
Yes I agree. For the time being, however, I think we can keep it as is until we come up with a suitable alternative.

Reply:
@zroubalik also, note that I've moved this code over to https://github.com/knative-sandbox/func-go which will change this PR to just be modifications to the templates.